config/build-cluster/cluster/200-memory-limitrange.yaml

```diff
@@ -22,5 +22,5 @@ spec:
     - default:
         memory: 8Gi
       defaultRequest:
-        memory: 4Gi
+        memory: 8Gi
```

Comment:
What's the reason for changing this?

Reply:
It used to be sidecar 4Gb, test container 4Gb, so actually a pod requests 8Gb. Now that we set sidecar to be real low, so we can make the real request of 8Gb

Reply:
I see, great improvements on resource utilization!